unstoppable-ios-app/domains-manager-ios/Services/Networking/NetworkService+ProfilesApi.swift

```diff
@@ -536,6 +536,12 @@ extension NetworkService {
     
     public func updateUserDomainNotificationsPreferences(_ preferences: UserDomainNotificationsPreferences,
                                                          for domain: DomainItem) async throws {
+        try await updateUserDomainNotificationsPreferences(preferences, for: domain, isRetryAfterSignatureFailed: false)
```

Comment:
`shouldRetryAfterSignatureFailed` ?

Reply:
Yes. There was update in how signature is retrieved to make it work with profile settings. Current users has old signature persisted that won't work. That will lead to error when user will try to block other user in chat for the first time (signature will be revoked, and on next attempt new extended signature will be generated and second request will work fine). This small recursion will handle this case.